include/swift/AST/Stmt.h

```diff
@@ -78,7 +78,6 @@ class alignas(8) Stmt {
   SourceLoc getEndLoc() const;
   
   SourceRange getSourceRange() const;
-  SourceLoc TrailingSemiLoc;
```

Comment:
Kind of a gray area, similar to escaping backticks, but we can keep track of things like this in the syntax tree.